rnndb/bus/pci.xml

```diff
@@ -489,6 +489,24 @@
 			<value value="2" name="2_5GT"/>
 		</bitfield>
 	</reg32>
+	<reg32 offset="0x080" name="WIDTH">
+		<bitfield low="0" high="7" name="UNK1_WIDTH"/>
+		<bitfield low="8" high="15" name="UNK2_WIDTH"/>
+	</reg32>
+	<reg32 offset="0x1c0" name="UNK1C0">
```

Comment:
Maybe you can call it CONFIG? That's the go-to term when we do not really know how to name it better.